src/minikube.ts

```diff
@@ -1,11 +1,13 @@
 import {addPath, getInput} from '@actions/core'
 import {exec} from '@actions/exec'
 import {downloadTool} from '@actions/tool-cache'
-import {platform as getPlatform} from 'os'
 import {mkdirP, mv} from '@actions/io'
+import {platform as getPlatform} from 'os'
 import {join} from 'path'
 
-export function setArgs(args: string[]) {
+import {restoreCaches, saveCaches} from './cache'
```

Comment:
is this the location on the GH VM that the caches will be stored ?

Reply:
No, this is the `cache.ts` file that contains the `restoreCaches` & `saveCaches` functions.